schema/cam/cam_schema_2-1-0.json

```diff
@@ -1,6 +1,18 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "$id": "https://Orange-OpenSource.github.io/its-client/schema/cam",
+  "$references": [
+   {
```

Comment:
Bad indentation; missing one leading space (indentation is 2-space).

Reply:
Ok, fixed